Lessons/Lesson-19-Css/html/index.html

```diff
@@ -73,23 +81,58 @@ <h3>Register</h3>
         body.className += " " + styleAttr;
     }
 
-    class MyGlobalScopeClass {
-        constructor() {
-          this.myProp = "prop";
-        }
+    function validate(evt) 
+    {
+        let inputNameResult =  ValidateInput('name');
+        if (!inputNameResult) return false;
 
-        getMyProp() {
-            return this.myProp;
-        }
+        let inputLastNameResult =  ValidateInput('last_name');
+        if (!inputLastNameResult) return false;
+
+        let checkboxEl = document.getElementById('accept');
+        if (!checkboxEl.checked)
+         {
+            alert("you must accept terms and conditions!");
+            return false;
+         }
 
-        setMyProp(val) {
-            this.myProp = val;
-        }
-    };
 
-    function validate(evt) {
-        return confirm("are you sure?");
+        return true;
+        //return confirm("are you sure?");
+    }
+
+    function ValidateInput(id)
```

Comment:
Please try to parameterize this function, so that different inputs can have different rules, consider to move it into ES6 class also